dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassPathBuilder.java

```diff
@@ -48,34 +48,47 @@ public ClassPathBuilder(DependencyGraphBuilder dependencyGraphBuilder) {
   }
 
   /**
-   * Builds a classpath from the transitive dependency graph of a list of artifacts.
-   * When there are multiple versions of an artifact in
-   * the dependency tree, the closest to the root in breadth-first order is picked up. This "pick
-   * closest" strategy follows Maven's dependency mediation.
+   * Builds a classpath from the transitive dependency graph from {@code artifacts}. When there are
+   * multiple versions of an artifact in the dependency tree, the closest to the root in
+   * breadth-first order is picked up. This "pick closest" strategy follows Maven's dependency
+   * mediation.
```

Comment:
except we don't handle version ranges :-(

Reply:
True. Maven dependency tree is more complex than we model our tree in this repository.